homeassistant/components/cert_expiry/config_flow.py

```diff
@@ -53,14 +54,16 @@ async def async_step_user(self, user_input=None):
         if user_input is not None:
             host = user_input[CONF_HOST]
             port = user_input.get(CONF_PORT, DEFAULT_PORT)
+            ca_cert = user_input.get(CONF_CA_CERT, "")
             await self.async_set_unique_id(f"{host}:{port}")
             self._abort_if_unique_id_configured()
 
             if await self._test_connection(user_input):
                 title_port = f":{port}" if port != DEFAULT_PORT else ""
                 title = f"{host}{title_port}"
                 return self.async_create_entry(
-                    title=title, data={CONF_HOST: host, CONF_PORT: port},
+                    title=title,
+                    data={CONF_HOST: host, CONF_PORT: port, CONF_CA_CERT: ca_cert},
```

Comment:
Perhaps it would be better to only save the `CONF_CA_CERT` key to the config entry if necessary.

Reply:
Was changed in the last commit